avm/res/cognitive-services/account/tests/e2e/max/main.test.bicep

```diff
@@ -88,12 +88,6 @@ module testDeployment '../../../main.bicep' = [for iteration in [ 'init', 'idem'
         storageAccountResourceId: diagnosticDependencies.outputs.storageAccountResourceId
         workspaceResourceId: diagnosticDependencies.outputs.logAnalyticsWorkspaceResourceId
       }
-      {
```

Comment:
intended?

Reply:
Yes, fixes test (see comment to your orginal PE test comment)